app/src/main/kotlin/com/wabadaba/dziennik/api/notification/NotificationHelper.kt

```diff
@@ -0,0 +1,35 @@
+package com.wabadaba.dziennik.api.notification
+
+import android.app.Notification
+import android.app.NotificationManager
+import android.app.NotificationChannel
+import android.os.Build
+import android.support.annotation.RequiresApi
+
+
+class NotificationHelper(private val notificationManager: NotificationManager) {
```

Comment:
Helper suffix doesn't convey any meaning. I see no point in having this class. Please move it's contents to LibrusGcmListenerService

Reply:
Changed suffix. Having this code in LibrusGcmListenerService would violate the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle)

Reply:
And this class allows us to simply reuse this notification channel.